### ISSUE TITLE
Redirect to correct page after upload

### ISSUE THREAD

**Issue by gmuraru** [vmck/acs-interface#153]
Redirect to the user submission page for the assignment that was uploaded.

**Comment by liisberg** [comment]
I can take of this if you will assign it to me :-)

**Comment by gmuraru** [comment]
Sure! It's yours for the taking :+1:

**Comment by liisberg** [comment]
@gmuraru 
The view `sub_for_subs` does not seem to exist.
Should uploads redirect to the view `submission_result`?

**Comment by gmuraru** [comment]
Yep - that one should be! Should I assign you to it? (sorry for the late reply)
